src/centralstore/incremental/NodeBlock.h

```diff
@@ -0,0 +1,56 @@
+#include <cstring>
```

Comment:
Lets add header

```
/**
Copyright 2020 JasminGraph Team
Licensed under the Apache License, Version 2.0 (the "License");
you may not use this file except in compliance with the License.
You may obtain a copy of the License at
    http://www.apache.org/licenses/LICENSE-2.0
Unless required by applicable law or agreed to in writing, software
distributed under the License is distributed on an "AS IS" BASIS,
WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
See the License for the specific language governing permissions and
limitations under the License.
 */
```

Reply:
Thanks , I have added the header in this commit https://github.com/miyurud/jasminegraph/pull/104/commits/2beb4b70c86e0062cab948abe425881b09496af2

Reply:
Ok